projects/1.20/assets/fungal-infection-spore/spore/lang/zh_cn.json

```
@@ -0,0 +1,422 @@
+  "entity.spore.gazenbreacher": "凝袭魔",
+  "entity.spore.sieger_tail": "强袭魔之尾",
+  "entity.spore.hindenburg": "菌械空堡",
+  "entity.spore.tumoroid_nuke": "瘤弹",
+  "entity.spore.licker": "舔舐者",
+  "entity.spore.plagued": "零号病人",
+  "entity.spore.lacerator": "突袭者",
+  "entity.spore.wave": "菌潮",
+  "entity.spore.illusion": "幻形魔",
+  "entity.spore.braurei": "酿毒体",
+  "entity.spore.verva": "保育体",
+  "entity.spore.delusioner": "妄念魔",
+  "entity.spore.gastgaber": "菌主",
+  "entity.spore.howitzer": "菌械重炮",
+  "entity.spore.howit_arm": "菌械重炮肢体",
+
+  "enchantment.spore.symbiotic_reconstitution.desc": "加强你和你的装备之间的共生纽带，处在共生状态下将修复其损伤。",
+  "enchantment.spore.cryogenic_aspect.desc": "使被攻击者感到刺骨的寒意，减慢其移动速度并造成冰冻伤害。",
+  "enchantment.spore.gastric_spewage.desc": "使血肉具有胆汁的特性，受害者将如同被胆汁喷淋。",
+  "enchantment.spore.corrosive_potency.desc": "使血肉具有腐蚀的特性，你的攻击将能够融化受害者的防御。",
+  "enchantment.spore.unwavering_nature.desc": "使血肉不再遗忘其寄染生命的原始欲望，即使那是它的主人",
+  "enchantment.spore.mutagenic_reactant.desc": "真菌的不稳定性旨在帮助其同族而不是其宿主。",
+
+
+  "entity.spore.thrown_tumor": "肿瘤",
+  "entity.spore.thrown_spear": "寄染长矛",
+  "entity.spore.acid_ball": "酸液球",
+  "entity.spore.bile": "胆汁",
+  "entity.spore.acid": "酸液",
+
+
+  "enchantment.spore.symbiotic_reconstitution": "共生重塑",
+  "enchantment.spore.cryogenic_aspect": "低温附加",
+  "enchantment.spore.gastric_spewage": "胃液反流",
+  "enchantment.spore.corrosive_potency": "腐蚀之效",
+  "enchantment.spore.unwavering_nature": "恒定自然",
+  "enchantment.spore.mutagenic_reactant": "诱变反应",
+
+
+  "effect.spore.mycelium_ef": "菌丝感染",
+  "effect.spore.stunt": "击晕",
+  "effect.spore.starvation": "极度饥饿",
+  "effect.spore.marker": "菌群凝视",
+  "effect.spore.corrosion": "盔甲腐蚀",
+  "effect.spore.symbiosis": "菌群共生",
+  "effect.spore.uneasy": "不安",
+  "effect.spore.madness": "疯狂",
+
+
+  "item.minecraft.splash_potion.effect.mycelium_potion": "喷溅型菌丝感染药水",
+  "item.minecraft.potion.effect.mycelium_potion": "菌丝感染药水",
+  "item.minecraft.lingering_potion.effect.mycelium_potion": "滞留型菌丝感染药水",
+  "item.minecraft.tipped_arrow.effect.mycelium_potion": "菌丝感染之箭",
+
+  "item.minecraft.splash_potion.effect.marker_potion": "喷溅型菌群凝视药水",
+  "item.minecraft.potion.effect.marker_potion": "菌群凝视药水",
+  "item.minecraft.lingering_potion.effect.marker_potion": "滞留型菌群凝视药水",
+  "item.minecraft.tipped_arrow.effect.marker_potion": "菌群凝视之箭",
+
+  "item.minecraft.splash_potion.effect.corrosion_potion": "喷溅型盔甲腐蚀药水",
+  "item.minecraft.potion.effect.corrosion_potion": "盔甲腐蚀药水",
+  "item.minecraft.lingering_potion.effect.corrosion_potion": "滞留型盔甲腐蚀药水",
+  "item.minecraft.tipped_arrow.effect.corrosion_potion": "盔甲腐蚀之箭",
+
+  "item.minecraft.splash_potion.effect.corrosion_potion_strong": "喷溅型强化盔甲腐蚀药水",
+  "item.minecraft.potion.effect.corrosion_potion_strong": "强化盔甲腐蚀药水",
+  "item.minecraft.lingering_potion.effect.corrosion_potion_strong": "滞留型强化盔甲腐蚀药水",
+  "item.minecraft.tipped_arrow.effect.corrosion_potion_strong": "强化盔甲腐蚀之箭",
+
+  "itemGroup.spore": "真菌感染 | 孢子",
+  "itemGroup.spore_t": "孢子 | 科技",

```

```suggestion
  "itemGroup.spore": "真菌感染：孢子",
  "itemGroup.spore_t": "孢子丨科技",
```
模组名用冒号
汉字丨gun3，仅用于分栏